packages/expo-gl/CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Changelog
 
-## master
+## 8.1.1
+
+### 🛠 Breaking changes
+
+### 🎉 New features
+
+### 🐛 Bug fixes
+
+- Allow createElement & unstable_createElement usage for web. ([#7543](https://github.com/expo/expo/pull/7995) by [@wood1986](https://github.com/wood1986))
+
+
+## 8.1.0
```

Comment:
```suggestion
## master

### 🐛 Bug fixes

- Allow createElement & unstable_createElement usage for web. ([#7543](https://github.com/expo/expo/pull/7995) by [@wood1986](https://github.com/wood1986))
```
No version is released so just add your entry to correct section under `## master` tag 😉